### PR TITLE
Rework of doc menus and it's locations

### PR DESCRIPTION
**Pull request by mpryc**
# Testing

https://pelorus--771.org.readthedocs.build/en/771/

Signed-off-by: Michal Pryc <mpryc@redhat.com>

@redhat-cop/mdt

### CODE REVIEW

File: docs/getting_started/QuickstartTutorial.md [hunk 1]
```diff
@@ -14,13 +14,13 @@ To do so, you will
 * Create new commits and issues
 * Watch as the metrics and trends change as new versions roll out**\***
 
-> **Note:** More information about the four key DORA metrics can be found in our [Outcomes documentation](./philosophy/outcomes/index.md).
+> **Note:** More information about the four key DORA metrics can be found in our [Outcomes documentation](#philosophy/outcomes/Overview/).
```
Comment (mateusoliveira43) [inline docs/getting_started/QuickstartTutorial.md:17]: this link did not work

File: mkdocs.yml [hunk 2]
```diff
@@ -51,25 +51,33 @@ markdown_extensions:
...
-    - NooBaa for Long Term Storage: Noobaa.md
+    - How Pelorus Can Help: philosophy/HowPelorusCanHelp.md
+  - Architecture: Architecture.md
   - Contribution:
     - Dev Guide: Development.md
     - Upstream Support: UpstreamSupport.md
 plugins:
   - social
+  - redirects:
+      redirect_maps:
```
Comment (mateusoliveira43) [inline mkdocs.yml:78]: what this do?

File: docs/getting_started/QuickstartTutorial.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Pelorus Demo
 
-For Pelorus installation to monitor your application's workflow, please skip this Demo and jump straight to the [Installation](Install.md) and [Configuration](configuration2.md) part of the documentation.
+For Pelorus installation to monitor your application's workflow, please skip this Demo and jump straight to the [Installation](../Installation/) and [Configuration](../configuration/PelorusCore/) part of the documentation.
```
[the remark this answers is not in the document]
Comment (mpryc) [reply docs/getting_started/QuickstartTutorial.md]: This PR is just to restructure, so no changes to the doc content.

File: docs/getting_started/QuickstartTutorial.md (hunk 1 above)
Comment (mpryc) [reply docs/getting_started/QuickstartTutorial.md]: Fixed, great catch. Actually mkdocs serve does not give warning as it's just link to local tag.

File: mkdocs.yml (hunk 2 above)
Comment (mpryc) [reply mkdocs.yml:78]: In our Pelorus Operator documentation in a bundle we have links to:

```
Pelorus Configuration Docs
https://pelorus.readthedocs.io/en/latest/Configuration/
```

```
https://pelorus.readthedocs.io/en/latest/Install/#configure-prometheus-retention
```

And few others. So to keep those links working we need to redirect to the new locations.

Comment (mateusoliveira43) [reply mkdocs.yml:78]: Oh now I understood. Agree with the workaround to do not touch operator, but I would open an issue to remove this when we update the operator

**Review by mateusoliveira43 — APPROVED** [review]

File: mkdocs.yml (hunk 2 above)
Comment (mpryc) [reply mkdocs.yml:78]: Created new issue to track this.

**Pull request merged**